frontend/src/components/PreviewReviewModal.tsx

```diff
@@ -0,0 +1,141 @@
+import {
+  Box,
+  HStack,
+  Image,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalHeader,
+  ModalOverlay,
+  Stack,
+  Tag,
+  Text,
+} from "@chakra-ui/react";
+import React from "react";
+
+/**
+ * The model defining the props for the Preview Review Modal component
+ */
+interface PreviewReviewModalProps {
+  /** The title of the book that is being reviewed */
+  title: string;
+  /** The subtitle of the book that is being reviewed */
+  subtitle: string;
+  /** The name of the author of the book */
+  writtenBy: string;
+  /** The name of the reviewer */
+  reviewedBy: string;
+  /** The publishes of the book */
+  publisher: string;
+  /** The ISBN number of the book */
+  isbn: string;
+  /** The type of the book */
+  bookType: string;
+  /** A description of the audience the book is recommended for */
+  ageDesciption: string;
+  /** The body of the review */
+  body: string;
+  /** The tags associated with the book */
+  tags: string[];
+  /** The URL of the cover image of the book */
+  coverUrl: string;
+}
+
+/**
+ * This component allows an admin to preview their review and get an understanding of
+ * what the subscribers of the review will see
+ */
+const PreviewReviewModal = ({
+  title,
+  subtitle,
+  writtenBy,
+  reviewedBy,
+  publisher,
+  isbn,
+  bookType,
+  ageDesciption,
+  body,
+  tags,
+  coverUrl,
+}: PreviewReviewModalProps): React.ReactElement => {
+  return (
+    <>
+      <ModalOverlay />
+      <ModalContent fontFamily="DM Sans">
+        <ModalHeader backgroundColor="#FFF8F3">
+          <HStack spacing="225px">
+            <HStack>
+              <Text fontSize={12} fontWeight={600}>
+                Previewing:
+              </Text>
+              <Text fontSize={12} fontWeight={400}>
+                Draft
+              </Text>
+            </HStack>
+            <Text fontSize={12} fontWeight={600}>
+              {title}
+            </Text>
+          </HStack>
+        </ModalHeader>
+        <ModalCloseButton />
+        <ModalBody margin={10}>
```

Comment:
Thoughts on if this bit should not be within the component and defined by the page itself?

Reply:
I feel like the <Modal> tag should be in the component, and isOpen and onClose are passed to the component as props

Reply:
done!